src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +14,22 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        try {
+            return factory.fromTransaction(session -> {
```

Comment:
It's better for learning purposes using methods for open session, begin transaction and commit or rollback